pyanaconda/modules/storage/partitioning/manual.py

```diff
@@ -87,25 +87,25 @@ def setup_kickstart(self, data):
 
         data_list = []
 
-        for mount_point in self.mount_points:
+        for request in self.requests:
             mount_data = data.MountData()
-            self._setup_mount_data(mount_data, mount_point)
+            self._setup_mount_data(mount_data, request)
```

Comment:
Is there a reason why the `mount_data` are not return value?

Reply:
Because you need `data` to create `mount_data` and `request` to set up `mount_data`. I am also not very happy about the methods `_setup_mount_data` and `_process_mount_data`. I am thinking about pasting their code back to the methods `setup_kickstart` and `process_kickstart`. What do you think?

Reply:
Maybe I'm missing something but why not change this to:
```
for request in self.requests:
    mount_data = self._setup_mount_data(data, request)

....
def _setup_mount_data(self, data, request):
    mount_data = data.MountData()
    <fill up mount data>
    return mount_data
```

The point is that you will not hide the side effect of the method. Before you didn't have a clue what it does.

Reply:
I have considered it, but I don't think that there is a good reason for keeping these methods at this moment. They are removed in https://github.com/rhinstaller/anaconda/commit/166cdf51109d2a07d02a581116ce7b60b253882b.